source/Activated.js

```diff
@@ -35,6 +35,13 @@ const Activated = ({
     setIsUnloading(true)
   }, [])
 
+  const handleEscClick = useCallback(e => {
+    e.preventDefault()
```

Comment:
I think `e.preventDefault()` should only be triggered if it's the escape key, don't you think? I reckon this should live inside the `if (...) { ... }`

Reply:
Applied!